platform/reference/migrations/4-0-migration.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Upgrade Guide from pre-v4.0 to v4.0+
+description: How to upgrade from pre-v4.0 to v4.0 release
+sidebar_position: 1
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+These docs cover upgrading from pre v4.0 to v4.0. Once you upgrade to v4.0+, you need to follow the regular [upgrade docs](../../administer/upgrade-migrate/upgrade). 
```

Comment:
The command
`vcluster platform start --upgrade --version=$PLATFORM_VERSION --values=vcluster-platform.yaml`

will not work if user is going from 34 to 4.0.0 to 4.0.1. It is missing `--namespace loft` field.

Should we call out this specific?

Reply:
I've updated the ugprade docs as well